src/test/java/com/ays/admin_user/controller/AdminUserControllerTest.java

```diff
@@ -0,0 +1,102 @@
+package com.ays.admin_user.controller;
+
+import com.ays.AbstractRestControllerTest;
+import com.ays.admin_user.model.AdminUser;
+import com.ays.admin_user.model.dto.request.AdminUserListRequest;
+import com.ays.admin_user.model.dto.request.AdminUserListRequestBuilder;
+import com.ays.admin_user.model.dto.response.AdminUsersResponse;
+import com.ays.admin_user.model.entity.AdminUserEntity;
+import com.ays.admin_user.model.entity.AdminUserEntityBuilder;
+import com.ays.admin_user.model.mapper.AdminUserEntityToAdminUserMapper;
+import com.ays.admin_user.model.mapper.AdminUserToAdminUsersResponseMapper;
+import com.ays.admin_user.service.AdminUserService;
+import com.ays.common.model.AysPage;
+import com.ays.common.model.dto.response.AysPageResponse;
+import com.ays.common.model.dto.response.AysResponse;
+import com.ays.common.model.dto.response.AysResponseBuilder;
+import com.ays.common.util.exception.model.AysError;
+import com.ays.util.AysMockMvcRequestBuilders;
+import com.ays.util.AysMockResultMatchersBuilders;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+import org.springframework.test.web.servlet.result.MockMvcResultHandlers;
+
+import java.util.List;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+class AdminUserControllerTest extends AbstractRestControllerTest {
+
+    @MockBean
+    private AdminUserService adminUserService;
+
+    private final AdminUserToAdminUsersResponseMapper ADMIN_USER_TO_ADMIN_USER_RESPONSE_MAPPER = AdminUserToAdminUsersResponseMapper.initialize();
+
+    private final AdminUserEntityToAdminUserMapper ADMIN_USER_ENTITY_TO_ADMIN_USER_MAPPER = AdminUserEntityToAdminUserMapper.initialize();
```

Comment:
This object should be declared as static.

Reply:
I just added `static `keyword to `AdminUserToAdminUsersResponseMapper` and `AdminUserEntityToAdminUserMapper `. Can we look through it again?